javascript/javascript2/week2/script.js

```diff
@@ -0,0 +1,122 @@
+const recipes = [
+    {
+        id: 1,
+        title: "Gløgg",
+        picture_url: "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
+        ingredients: [
+          { NAME: "Orange zest", AMOUNT: "0.5" },
```

Comment:
```suggestion
          { name: "Orange zest", amount: "0.5" },
```
This will be more pleasing to the eye